.github/workflows/sycl-linux-build.yml

```diff
@@ -247,13 +250,13 @@ jobs:
         retention-days: ${{ inputs.retention-days }}
 
     - name: Copy toolchain
-      if: ${{ always() && !cancelled() && steps.build.conclusion == 'success' }}
+      if: ${{ inputs.e2e_binaries_artifact && always() && !cancelled() && steps.build.conclusion == 'success' }}
       # We must have the compiler in the same location as it will be in the E2E
       # run-tests job.
       run: cp -r $GITHUB_WORKSPACE/build/install $GITHUB_WORKSPACE/toolchain
 
     - name: Source OneAPI TBB vars.sh
-      shell: bash
```

Comment:
Don't we need bash to setup OneAPI environment?

Reply:
Not for the reasons I expected, but we do need it for now (I think it's `source ` vs `. ` here).